src/layouts/ticketFormLayout.js

```diff
@@ -0,0 +1,364 @@
+export default [
```

Comment:
I would move this file into `/tests/DemoState` as it is only needed for tests and storybook and would need to be provided when using this library